content/git/concepts/clone/clone.md

```diff
@@ -12,24 +12,24 @@ CatalogContent:
   - 'learn-the-command-line'
 ---
 
-In Git, cloning means copying an existing remote repository to your current directory.
+In Git, cloning means copying an existing remote repository to the current directory.
 
-Some of the basic Git commands have been performed only on a local branch, that is, a branch that exists on your computer. Here's a command that will help you interact with a remote branch &mdash; a branch on a server that can be accessed through the internet.
+Some of the basic Git commands have been performed only on a local branch, that is, a branch that exists on a local machine. Here's a command that will interact with a remote branch &mdash; a branch on a server that can be accessed through the internet.
```

Comment:
```suggestion
Some of the basic Git commands have been performed only on a local branch, that is, a branch that exists on a local machine. Here's a command that will interact with a remote branch &mdash; a branch on a server that can be accessed through the internet.
```

Would it be more accurate to say "repository" instead of "branch"? Or maybe "clone or branch" if `git clone` can clone from a specific branch?